<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1350</width>
    <height>900</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Analizador de Gramáticas</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../../images/Qt.ico</normaloff>../../images/Qt.ico</iconset>
  </property>
  <property name="tabShape">
   <enum>QTabWidget::Rounded</enum>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_3">
    <item row="0" column="0">
     <widget class="QTabWidget" name="tabWidget">
      <property name="font">
       <font>
        <family>Tahoma</family>
        <pointsize>12</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tabGrammar">
       <attribute name="title">
        <string>Gramática</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0">
         <widget class="QPlainTextEdit" name="textEditGrammar">
          <property name="font">
           <font>
            <family>Consolas</family>
            <pointsize>28</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="plainText">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tabResults">
       <attribute name="title">
        <string>Resultados</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <widget class="QWebView" name="textResults">
          <property name="url">
           <url>
            <string>about:blank</string>
           </url>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1350</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>Archivo</string>
    </property>
    <addaction name="separator"/>
    <addaction name="actionNewGrammar"/>
    <addaction name="separator"/>
    <addaction name="actionLoadGrammar"/>
    <addaction name="separator"/>
    <addaction name="actionSaveGrammar"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuAcerca_de">
    <property name="title">
     <string>Acerca de</string>
    </property>
    <addaction name="actionAboutAuthors"/>
    <addaction name="actionAboutGrammarAnalyser"/>
   </widget>
   <widget class="QMenu" name="menuAnalisis">
    <property name="title">
     <string>Análisis</string>
    </property>
    <addaction name="actionAnalyse_2"/>
    <addaction name="actionQuickAnalyse_2"/>
   </widget>
   <widget class="QMenu" name="menuAyuda">
    <property name="title">
     <string>Ayuda</string>
    </property>
    <addaction name="actionHelp_2"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuAnalisis"/>
   <addaction name="menuAcerca_de"/>
   <addaction name="menuAyuda"/>
  </widget>
  <widget class="QStatusBar" name="statusbar">
   <property name="font">
    <font>
     <family>Tahoma</family>
     <pointsize>12</pointsize>
    </font>
   </property>
  </widget>
  <action name="actionNewGrammar">
   <property name="icon">
    <iconset>
     <normaloff>../../images/selection-input.png</normaloff>../../images/selection-input.png</iconset>
   </property>
   <property name="text">
    <string>Nueva gramática</string>
   </property>
   <property name="statusTip">
    <string>Nueva Gramática</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionSaveGrammar">
   <property name="icon">
    <iconset>
     <normaloff>../../images/disk.png</normaloff>../../images/disk.png</iconset>
   </property>
   <property name="text">
    <string>Guardar gramática</string>
   </property>
   <property name="statusTip">
    <string>Guardar Gramática</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionLoadGrammar">
   <property name="icon">
    <iconset>
     <normaloff>../../images/blue-folder-horizontal-open.png</normaloff>../../images/blue-folder-horizontal-open.png</iconset>
   </property>
   <property name="text">
    <string>Cargar gramática</string>
   </property>
   <property name="statusTip">
    <string>Cargar Gramática</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="icon">
    <iconset>
     <normaloff>../../images/cross-circle.png</normaloff>../../images/cross-circle.png</iconset>
   </property>
   <property name="text">
    <string>Salir</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+F4</string>
   </property>
  </action>
  <action name="actionHelp">
   <property name="icon">
    <iconset>
     <normaloff>../../images/question.png</normaloff>../../images/question.png</iconset>
   </property>
   <property name="text">
    <string>Ayuda</string>
   </property>
   <property name="statusTip">
    <string>¿Cómo usar?</string>
   </property>
   <property name="shortcut">
    <string>F1</string>
   </property>
  </action>
  <action name="actionHelp_2">
   <property name="icon">
    <iconset>
     <normaloff>../../images/question.png</normaloff>../../images/question.png</iconset>
   </property>
   <property name="text">
    <string>Ayuda</string>
   </property>
   <property name="shortcut">
    <string>F1</string>
   </property>
  </action>
  <action name="actionAboutGrammarAnalyser">
   <property name="icon">
    <iconset>
     <normaloff>../../images/lifebuoy.png</normaloff>../../images/lifebuoy.png</iconset>
   </property>
   <property name="text">
    <string>Acerca de Analizador de Gramáticas</string>
   </property>
  </action>
  <action name="actionSaveGrammarAs">
   <property name="icon">
    <iconset>
     <normaloff>../../images/disk--pencil.png</normaloff>../../images/disk--pencil.png</iconset>
   </property>
   <property name="text">
    <string>Guardar gramática como ...</string>
   </property>
   <property name="statusTip">
    <string>Guardar Gramática Como</string>
   </property>
  </action>
  <action name="actionAnalyse">
   <property name="icon">
    <iconset>
     <normaloff>images/control.png</normaloff>images/control.png</iconset>
   </property>
   <property name="text">
    <string>Analizar</string>
   </property>
   <property name="toolTip">
    <string>Analizar</string>
   </property>
   <property name="statusTip">
    <string>Analizar</string>
   </property>
   <property name="font">
    <font>
     <pointsize>8</pointsize>
    </font>
   </property>
   <property name="shortcut">
    <string>F5</string>
   </property>
  </action>
  <action name="actionQuickAnalyse">
   <property name="icon">
    <iconset>
     <normaloff>images/controlx2.png</normaloff>images/controlx2.png</iconset>
   </property>
   <property name="text">
    <string>Análisis rápido</string>
   </property>
   <property name="toolTip">
    <string>Análisis rápido</string>
   </property>
   <property name="statusTip">
    <string>Análisis rápido</string>
   </property>
   <property name="shortcut">
    <string>F9</string>
   </property>
  </action>
  <action name="actionAnalyse_2">
   <property name="icon">
    <iconset>
     <normaloff>../../images/control.png</normaloff>../../images/control.png</iconset>
   </property>
   <property name="text">
    <string>Análisis de la gramática</string>
   </property>
   <property name="shortcut">
    <string>F5</string>
   </property>
  </action>
  <action name="actionQuickAnalyse_2">
   <property name="icon">
    <iconset>
     <normaloff>../../images/controlx2.png</normaloff>../../images/controlx2.png</iconset>
   </property>
   <property name="text">
    <string>Análisis rápido de la gramática</string>
   </property>
   <property name="shortcut">
    <string>F9</string>
   </property>
  </action>
  <action name="actionAboutAuthors">
   <property name="icon">
    <iconset>
     <normaloff>../../images/lifebuoy.png</normaloff>../../images/lifebuoy.png</iconset>
   </property>
   <property name="text">
    <string>Acerca de los Autores</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QWebView</class>
   <extends>QWidget</extends>
   <header>QtWebKitWidgets/QWebView</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
